pwm/ad_notify_passwd_expire.sls

```diff
@@ -0,0 +1,45 @@
+{%- if salt.pillar.get('pwm:lookup:ad_notify_passwd_expire') %}
+passwd_notify_install_packages:
+  pkg.installed:
+    - names:
+      - git
+      - php
+      - php-ldap
+      - cyrus-sasl-plain
+
+gitproj:
+  git.latest:
+    - name: https://github.com/ewierschke/ad-password-expire-notify
+    - target: /usr/local/bin/ad-password-expire-notify
+    - rev: pwm
+
+env_adjust_mode:
+  file.managed:
+    - name: /usr/local/bin/ad-password-expire-notify/envadjust.sh
+    - mode: 700
+    - replace: False
+
+execute_env_adjust:
+  cmd.run:
+    - name: /usr/local/bin/ad-password-expire-notify/envadjust.sh
+
+create_daily_notify_cronjob:
+  cmd.run:
```

Comment:
Use the cron state, https://docs.saltstack.com/en/latest/ref/states/all/salt.states.cron.html